cli/remotebazel/remotebazel.go

```diff
@@ -561,13 +559,14 @@ func Run(ctx context.Context, opts RunOpts, repoConfig *RepoConfig) (int, error)
 		req.Timeout = timeout.String()
 	}
 
+	log.Printf("\nWaiting for available remote runner...\n")
 	rsp, err := bbClient.Run(ctx, req)
 	if err != nil {
 		return 0, status.UnknownErrorf("error running bazel: %s", err)
 	}
 
 	iid := rsp.GetInvocationId()
-	log.Debugf("Invocation ID: %s", iid)
+	log.Printf("\nStreaming remote bazel logs to: %s/invocation/%s", *remoteRunner, iid)
```

Comment:
IIRC the remote_runner flag is a gRPC target? if so, this won't necessarily be a valid http URL that people can click. We could add a secondary flag like `remote_runner_results_url` which defaults to `https://app.buildbuddy.io/invocation/` - similar to Bazel's `bes_results_url` flag except with a default value set to match `remote_runner`.

Reply:
Since the `ci_runner` knows the source of truth, should we make the CI runner's first line of log output be something like "Streaming (workflow |remote bazel) logs to <url>"?

Reply:
Oh good idea - changed